languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -4248,6 +4248,14 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             <!--
            MARCOAGPINTO inserted global antipatterns for all subrules on top using the logic (2-MAR-2023+)
       -->
+            <antipattern>
+                <token>a</token>
+                <token>nós</token>
+                <token postag="AQ..S.+|N..S.+" postag_regexp="yes"/>
```

Comment:
You could also add an optional determiner before the noun/adjective here.

Reply:
@susanaboatto 

Could you explain?

Do you mean something like this?:

```
            <antipattern>
                <token>a</token>
                <token postag="PP..P.+" postag_regexp="yes"/>
                <token postag="AQ..S.+|N..S.+" postag_regexp="yes"/>
                <token postag='RM|RG|CS|CC|SPS00|(SPS00)?:[DP]...[SN].+|VM...S.+' postag_regexp='yes'/>
                <example>Assim que partimos, juntou-se a nós Jesus com pequeno grupo de seguidores.</example>
                <example>A nós Jesus disse estar cansado de pregar.</example>
            </antipattern>
```

Reply:
>            <antipattern>
>                 <token>a</token>
>                 <token>nós</token>
>                 <token min="0" postag="D.+" postag_regexp="yes"/>
>                 <token postag="AQ.+|N.+" postag_regexp="yes"/>
>                 <token postag='RM|RG|CS|CC|SPS00|(SPS00)?:[DP]...[SN].+|VM...S.+' postag_regexp='yes'/>
>                 <example>Assim que partimos, juntou-se a nós o menino com os presentes.</example>
>             </antipattern>

Reply:
I hadn't checked the main rule though, let's leave it as it is.